medusa/metadata/wdtv.py

```diff
@@ -162,12 +150,11 @@ def get_season_poster_path(show_obj, season):
                 break
 
         if not season_dir:
-            logger.log(u'Unable to find a season directory for season {season_num}'.format
-                       (season_num=season), logger.DEBUG)
+            log.debug(u'Unable to find a season directory for season {0}', season)
             return None
 
-        logger.log(u'Using {path}/folder.jpg as season dir for season {season_num}'.format
-                   (path=season_dir, season_num=season), logger.DEBUG)
+        log.debug(u'Using {path}/folder.jpg as season dir for season {x}',
+                  {'path': season_dir, 'x': season})
```

Comment:
I chose `x` for things like that because its a common convention to represent a number as `x` and as `x` isnt actually a variable it doesnt pollute the local namespace with one-letter variables

also imo its clearer to read then `with {number} seeders` or `with {seeders} seeders` or in this case `for season {season}`

as proof that its not actually a variable try this:  `'This is a {variable test}'.format(**{'variable test': 'success!'})`
it works even though `variable test` isnt a valid variable due to the space